src/workflows/dags/forecast_prediction_jobs.py

```
@@ -3,7 +3,7 @@
 from datetime import datetime
 
 from airflow.decorators import dag, task
-from dateutil.relativedelta import relativedelta
+from datetime import datetime, timedelta, timezone
```

Ruff (0.8.2)

Redefinition of unused `datetime` from line 3

Remove definition: `datetime`

(F811)